packages/data/src/api/accountsByPublicKey/README.md

```diff
@@ -0,0 +1,35 @@
+[//]: # `{ "title": "accountsByPublicKey", "order": "1.0.18" }`
+
+# accountsByPublicKey
+
+{% hint style="warning" %}
+
+This is a work in progress, please reach out to us on [Telegram](https://t.me/mintdev) for support.
+
+For the most reliable data, reference our [existing graphql docs](https://docs.mintbase.io/dev/read-data/mintbase-graph).
+
+{% endhint %}
+
+Returns accounts for which `publicKey` is a full access key.
+
+### accountsByPublicKey(publicKey: string): Promise<ParsedDataReturn<string[]>>
+
+Example:
+
+
+
+{% code title="accountsByPublicKey.ts" overflow="wrap" lineNumbers="true" %}
+
+```typescript
```

Comment:
FYI: I believe you can also go with just ts here:

```suggestion
```ts
```

Reply:
Yup, but this is consistent with the rest of documentation (I just copied the template from another README)